src/ui/components/Navigation/components/BatchPanel/BatchPanel.tsx

```diff
@@ -12,21 +12,32 @@ const i18n = I18n.keyset('component.navigation.view');
 
 export type BatchPanelProps = {
     count: number;
+    countForMove?: number;
+    countForDelete?: number;
     onAction: (action: BatchAction) => void;
     onClose: () => void;
     className?: string;
 };
 
-export const BatchPanel = ({count, className, onAction, onClose}: BatchPanelProps) => {
+export const BatchPanel = ({
```

Comment:
Let's fork `BatchPanel` for collections? Icon for folder navigation shouldn't be changed. Folder and collection navigation have difference, I think it would be easier to maintain two different components.

<img width="775" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/c8f74c2f-ed96-4864-b6a6-7b7d4c1f46a0">

And let's hide `Selected objects` block in forked component:

<img width="456" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/04c1a8f4-debc-40f9-9181-54bf92992eea">

Reply:
yes, it will be better, added new component CollectionBatchPanel